extensions/udapdr/scripts/dsp_utilities/evaluate_beir.py

```diff
@@ -0,0 +1,79 @@
+
+import pandas as pd
+import numpy as np
+import json
+from sklearn.metrics import ndcg_score
+from tqdm import tqdm
+import ast
+
+def evaluate_beir(distilled_ranking, chosen_BEIR_set, chosen_BEIR_type, chosen_LoTTE_split=None, chosent_LoTTE_type=None, chosent_LoTTE_set=None):
```

Comment:
Why `chosent_LoTTE_type` instead of `chosen_LoTTE_type`? Also why are there `LoTTE` variables in the `evaluate_beir` function?